src/components/error-summary/index.md.njk

```diff
@@ -11,7 +11,7 @@ layout: layout-pane.njk
 
 Use this component at the top of a page to summarise any errors a user has made.
 
-When a user makes an error, you must show both an error summary and an [error message](../error-message/) next to each field that contains an error.
+When a user makes an error, you must show both an error summary and an [error message](../error-message/) next to each question that contains an error.
```

Comment:
It'll be the answer, rather than the question, that contains an error - so I'd change the wording to:

```suggestion
When a user makes an error, you must show both an error summary and an [error message](../error-message/) next to each answer that contains an error.
```